components/Header/Header.jsx

```diff
@@ -141,7 +141,7 @@ export default ({
   openMenu,
   params,
   location,
-}) => (<div>
```

Comment:
Can we use css files when we can (so there is one source of truth), and can we space it like html instead of like curly brackets?